docs/src/tutorials/conversions_views.md

```diff
@@ -507,3 +507,33 @@ Base.ReshapedArray{RGB{Normed{UInt8,8}},3,Base.ReinterpretArray{RGB{Normed{UInt8
 While there is little or no performance cost to making use of
 JuliaImage's convenient views, sometimes the types can get
 complicated!
+
+## Preserving colorant type while conversion
```

Comment:
Let's instead put this info in `arrays_colors.md` right before the `A consistent scale for floating-point and "integer" colors: fixed-point numbers` section.